src/StoryDeck.elm

```diff
@@ -287,4 +300,33 @@ decks =
               }
             ]
       }
+    , { id = 4
+      , title = StoryFourTitle
+      , teaser = StoryFourTeaser
+      , teaserImgPath = AssetPath "story_images/story4-thumbnail.jpg"
+      , teaserImgAltText = StoryFourTeaserImageAlt
+      , relatedInfo = relatedInfo 3
```

Comment:
should this be 3 or 4? the story with id = 3 has `relatedInfo 3` too.

Reply:
Nice.